binding_generator.py

```diff
@@ -402,6 +408,8 @@ def generate_bindings(api_filepath, use_template_get_node, bits="64", precision=
     generate_builtin_bindings(api, target_dir, real_t + "_" + bits)
     generate_engine_classes_bindings(api, target_dir, use_template_get_node)
     generate_utility_functions(api, target_dir)
+    if godot_repo != "":
+        generate_compat_includes(godot_repo, output_dir, target_dir)
```

Comment:
So, this is modifying the generated files afterwards? Why not just make this part of generating the files in the first place?

Reply:
There are a lot of functions that create the file. Also this is how it was previously as I created the file in another folder. I can update if you want, but that will polute the smaller functions (that generate file in the first place)

Reply:
Personally, I think we should update the functions that generate the files in the first place

Reply:
Ok, well, got it working now all around. If you want this change, I can do that instead, but first let me bring some counter arguments I guess. If in future we don't need this tight integration anymore, it's easier to remove. All the code for compat is in one function. If there are bugs/issues with it, its one function to debug, and not multiple functions.
Downsides to having it separate is that people might not test it as often and it might break, though since in other functions it will be under if's anyway, won't be much tighter integrated anyway.

So yeah, if you think it's better to integrate them in functions, will update again the code. Just double checking.

Reply:
I really do think we should integrate this into the functions generating the files in the first place. But I guess that's just my opinion? If you can get other GDExtension team members on-board, I'd be fine to go with the consensus, but I personally think it's silly to modify files after the fact, when it's the same script that's generating them in the first place.

Reply:
Who are other gdextension members?

Reply:
See https://godotengine.org/teams/#scripting